src/components/MenuAccount.vue

```diff
@@ -89,10 +92,10 @@ function handleAction(e) {
       </template>
     </BaseMenu>
   </div>
+
   <teleport to="#modal">
-    <ModalEmailSubscription
+    <ModalEmailHOC
```

Comment:
We should use `showModalName`

Reply:
You only use it component once, so I don't really see the need for the HOC component

Reply:
No, the `modalEmailSubscriptionOpen` we should use `showModal<Name>` from now on everywhere. I used `modal<Name>Open` in the past, but it's less readable.